FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -28,11 +24,13 @@
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__) \
 { \
+NSString *envReferenceImageDirectory = [NSProcessInfo processInfo].environment[@"FB_REFERENCE_IMAGE_DIR"]; \
```

Comment:
What do you mean with "will support". In a future release?
Swift is not the only problem here. When using a frameworks, the pre-defined macro from this library is `$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages` instead of `/Users/me/code/SomeProjectTests/ReferenceImages` even when using Objective-C. The only way it works is with static libraries.